src/main/java/io/phdata/streamliner/schemadefiner/model/Snowflake.java

```diff
@@ -38,6 +38,13 @@
   public SnowflakeDatabase reportingDatabase;
   public String stageName;
   public IngestFileFormat fileFormat;
+  /* This is used in case user wants to remove substrings from source table name.
```

Comment:
I think there could be several strategies.

1. Search/Replace (The strategy were we remove the prefix can be represented more generically as search/replace)
2. Add postfix
3. Add prefix

Similar to IngestFileFormat, how about we add a class TableNameStrategy and then have a two options:

1. As-is => does nothing
2. Search/Replace => pattern (java regex) and replacement (the replacement string)